web/src/findLogLine.ts

```diff
@@ -0,0 +1,19 @@
+const LINE_ID_ATTR_NAME = "data-lineid"
+
+export default function findLogLineID(
+  el: HTMLElement | Node | null
+): string | null {
+  if (el === null) {
+    return null
+  }
+
+  if ((el as HTMLElement).getAttribute(LINE_ID_ATTR_NAME)) {
```

Comment:
i think you need to check that getAttribute exists, right? the `as` doesn't do any runtime checks

Reply:
Yeah I'm trying to figure out the idiomatic way to do this in typescript...